stanford_news.install

```diff
@@ -15,4 +15,6 @@ function stanford_news_install() {
   if ($tax_view = View::load('taxonomy_term')) {
     $tax_view->disable()->save();
   }
+
+  \Drupal::service('theme_installer')->install(['stable']);
```

Comment:
Why is this needed?

Reply:
https://github.com/SU-SWS/stanford_news/blob/8.x-2.x/templates/field/field--node--su-news-dek--stanford-news.html.twig#L1

Reply:
the `standard` profile no longer installs the `stable` them OOTB. so we have to explicitly make sure its enabled. and unfortunately we can't do that via the info.yml file.

Reply:
Another option is to refactor the template and remove that `stable` dependency

Reply:
Hmm. I wonder if we can just drop the @stable namespace in the extend and go with simply `{% extends 'field.html.twig' %}`

Reply:
Perhaps try that instead.

Reply:
after looking at that template, it was completely unnecessary when a preprocess hook can add the classes just the same. I've removed the template and refactored it into a hook. I'd like to come back to the other templates in this module at some point. I feel like they were just put in to get by at the time and we can improve on them.

Reply:
Sounds like a plan to me.